account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -74,20 +73,12 @@ def _default_sequence_ids(self):
     update_default_taxes = fields.Boolean(
         help="Update default taxes applied to local transactions",
     )
-    default_sale_tax_id = fields.Many2one(
```

Comment:
We are losing this feature (and all the related), that is very interesting. Maybe we can to populate CoA data and offer here a selection from that data?

Reply:
It is not possible to do that because among other things, the `account.tax` records do not exist yet. These changes are explained in the description of the PR and the currently existing approach.

Reply:
I'm not saying to show a many2one, but a selection with the populated possible taxes brought by populating CoA data, and then after the creation, select the corresponding created account.tax.